datalake-spark31/src/main/scala/bio/ferlab/datalake/spark3/transformation/PBKDF2.scala

```diff
@@ -0,0 +1,55 @@
+package bio.ferlab.datalake.spark3.transformation
+
+import com.roundeights.hasher.Implicits._
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions._
+
+import java.time.LocalDateTime
+import scala.language.postfixOps
+
+/*
+List(
```

Comment:
place the cursor right above 'case class PBKDF2 and type /** then type ENTER
now use this to add you comments about the class
it should be green on IntelliJ instead of grey

Reply:
done